boltdesignsystem/bolt
# DS-433: Remove CSS Nano from our dependencies

## Jira

https://pegadigitalit.atlassian.net/browse/DS-433

## Summary

Remove CSS Nano from Bolt Design System dependencies.

## Details

Remove CSS Nano from `package.json` and `yarn.lock`

## How to test

1. Run this code locally
2. Run `yarn clean`
3. Run `yarn setup`
4. Run `yarn start`
5. Check if everything looks good
6. Run `yarn build`
7. Confirm that build process finish successfully
